subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/ToolingApiClasspathIntegrationTest.groovy

```diff
@@ -30,7 +30,7 @@ class ToolingApiClasspathIntegrationTest extends AbstractIntegrationSpec {
         then:
         resolve.classpath.size() == 2
         resolve.classpath.any {it.name ==~ /slf4j-api-.*\.jar/}
-        resolve.classpath.find { it.name ==~ /gradle-tooling-api.*\.jar/ }.size() < 1.7 * 1024 * 1024
+        resolve.classpath.find { it.name ==~ /gradle-tooling-api.*\.jar/ }.size() < 1.8 * 1024 * 1024
```

Comment:
I'm not sure if this is the right "fix" for this test...
My understanding is that this test checks the size of the jar and it is normal to have it increased from time to time since code is added to it. Is that right?

Reply:
Yes that's right - This is just to make sure it is a minified JAR and we don't accidentally package unnecessary stuff in there. The failure serves as a reminder to ask "Did I really add that much code?"